### ISSUE TITLE
Missing ferncolorized texture in minecraft domain - incorrect model referencing a texture without a modid?

### ISSUE THREAD

**Issue by alcatrazEscapee** [TerraFirmaCraft/TFC-Assets#22]
Idk what this is so I'll leave it here. Log shows missing texture from minecraft domain. Probably an error with a TFC texture being refrenced via an invalid / missing mod id
```
[13:18:43] [Client thread/ERROR] [FML.TEXTURE_ERRORS]:     The missing resources for domain minecraft are:
[13:18:43] [Client thread/ERROR] [FML.TEXTURE_ERRORS]:       textures/plants/ferncolorized.png
```

**Comment by alcatrazEscapee** [comment]
Found the file: `assets\tfc\models\block\plants\lady_fern.json`
```
"textures": {
    "0": "plants/FernColorized",
    "1": "tfc:blocks/plants/lady_fern/fern_single",
    "particle": "tfc:blocks/plants/lady_fern/fern_single"
  },
```

**Comment by alcatrazEscapee** [comment]
Fixed by f43ca8a04d3d71c3da594605683bd344c07fc04d
